service/defaultcomponents/default_extensions_test.go

```diff
@@ -25,26 +25,47 @@ import (
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/extension/healthcheckextension"
+	"go.opentelemetry.io/collector/extension/pprofextension"
+	"go.opentelemetry.io/collector/extension/zpagesextension"
+	"go.opentelemetry.io/collector/testutil"
 )
 
 func TestDefaultExtensions(t *testing.T) {
 	allFactories, err := Components()
 	require.NoError(t, err)
 
 	extFactories := allFactories.Extensions
+	endpoint := testutil.GetAvailableLocalAddress(t)
+	port := testutil.GetAvailablePort(t)
 
 	tests := []struct {
 		extension   configmodels.Type
 		getConfigFn getExtensionConfigFn
 	}{
 		{
 			extension: "health_check",
+			getConfigFn: func() configmodels.Extension {
+				cfg := extFactories["health_check"].CreateDefaultConfig().(*healthcheckextension.Config)
+				cfg.Port = port
```

Comment:
Can you file an issue for the health_check extension to also accept `endpoint`?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/issues/2764